Makefile.defs

```diff
@@ -463,7 +463,8 @@ ifneq (,$(findstring march=arm,$(CFLAGS)))
 	# We're on ARM
 	ifeq (,$(findstring fpu=neon,$(CFLAGS)))
 		# We don't support NEON
-		undefine WANT_SIMD
+		# NOTE: undefine is new to GNU make 3.8...
```

Comment:
Umm where do we come across this make from, what, 2010? Older? >_>

Reply:
Oh, the Android TC? Makes sense. They also use some shell scripting methods that were deprecated around the year I was born, so… :-P

Reply:
Android, apparently ;D

Reply:
That's should say gmake 3.82, but, still ;p.